apps/web-portal/lib/preview/makePreview.mjs

```diff
@@ -0,0 +1,9 @@
+import {PreviewList} from './PreviewList.mjs'
+
+export function makePreview ({makeRunner}) {
+   return {
+     list: makeRunner(PreviewList, {
+       toParams: (req) => ({...req.query})
```

Comment:
maybe add as example response func?

Reply:
if you are talking about samples for the usage of 'makeRunner', then it's a valid concern for documentation for 'utils-use-cases' readme, plz comments/suggestions to that file so I can extract them as tasks